src/runtime/methodwrapper.cs

```diff
@@ -13,7 +13,6 @@ internal class MethodWrapper
         public IntPtr mdef;
         public IntPtr ptr;
         private bool _disposed = false;
-
```

Comment:
if you add this empty line back in you remove an entire file from the squash-commit.